scripts/examples/gn_silabs_example.sh

```diff
@@ -272,7 +272,9 @@ else
     fi
 
     # 917 exception. TODO find a more generic way
-    if [ "$SILABS_BOARD" == "BRD4325B" ]; then
+    if [ "$SILABS_BOARD" == "BRD4325B" ] || [ "$SILABS_BOARD" == "BRD4325C" ]; then
+        # TODO: remove this flag once ota is supported
+        optArgs+="chip_enable_ota_requestor=false "
```

Comment:
Hit this issue two when trying to build 917 soc. Since this is a large PR and I needed the 917 soc to build now, I made a small PR to fix a bunch of current build issues.
I believe it is better to just remove setting the define in `examples/platform/silabs/SiWx917/BUILD.gn`
see what I did for this in this [PR](examples/platform/silabs/SiWx917/BUILD.gn)

Reply:
The PR got merged, You don't need this change here in `gn_silas_example.sh` because it is addressed with this 

https://github.com/project-chip/connectedhomeip/blob/master/examples/platform/silabs/SiWx917/BUILD.gn#L188

Reply:
removed below 2 lines

 # TODO: remove this flag once ota is supported
        optArgs+="chip_enable_ota_requestor=false "